app/models/global_state.rb

```diff
@@ -58,4 +61,16 @@ def self.valid_new_incident_years
     # for last year iff submission hasn't closed out for the year.
     submission_closed_for_year? ? [Time.current.year] : [Time.current.year - 1, Time.current.year]
   end
+
+  def self.maintenance_mode?
+    instance.maintenance_mode
```

Comment:
I don't think that a migration or a fallback is necessary, actually. If `maintenance_mode` isn't defined it'll just be `nil`, which is falsey, which is the default we want anyway.

I could write `instance.maintenance_mode || false` but that seems redundant.